libgalois/include/katana/analytics/ClusteringImplementationBase.h

```diff
@@ -307,7 +307,7 @@ struct ClusteringImplementationBase {
     return mod;
   }
 
-  template <typename EdgeWeightType>
+  template <typename EdgeWeightType, typename NodePropTy>
```

Comment:
nit: `NodePropType` or `EdgeWeightTy` or I think the predominate style in the rest of the codebase is just `EdgeWeight` `NodeProp`